tests/designs/avalon_streaming_module/Makefile

```
@@ -20,7 +20,7 @@ COCOTB?=$(WPWD)/../../..
 
 VERILOG_SOURCES = $(COCOTB)/tests/designs/avalon_streaming_module/avalon_streaming.sv
```

If the goal is to eliminate uses of $(COCOTB), should this be:
```suggestion
VERILOG_SOURCES = $(WPWD)/avalon_streaming.sv
```

---

See https://github.com/cocotb/cocotb/pull/1517#issuecomment-601751008.

---

If these are no longer global, shouldn't we change `COCOTB?=` to `COCOTB:=` in those files?
Allowing users to run the makefile from one install to run the tests in a different install (by setting COCOTB) doesn't make any sense.

---

Ah, I see with your new commit message it's pretty clear that this change was out of scope, I was still reading the PR title

---

>If these are no longer global, shouldn't we change COCOTB?= to COCOTB:= in those files?
Allowing users to run the makefile from one install to run the tests in a different install (by setting COCOTB) doesn't make any sense.
That's a good follow on commit. I can put that in this PR if you want.

---

Nah, I think fine to leave it till another PR. I'd still prefer we eliminate it entirely, as the current structure makes renaming the folder really hard. `../../../../path/to/current/folder` is just a fragile way to spell `.`.